components/Api/Stability/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { PropsWithChildren } from 'react';
+
+type Props = PropsWithChildren<{ stability: number }>;
```

Comment:
Type should be close to the component 👀 

Also...

```suggestion
type StabilityProps = PropsWithChildren<{ stability: number }>;
```